src/suvela/exams/Exam1.java

```diff
@@ -0,0 +1,45 @@
+package suvela.exams;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import suvela.basic.RequiredActions;
+
+public class Exam1 {
+
+	public void compareStrings(WebDriver driver) {
+		driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+		driver.findElement(By.linkText("Demo Tables")).click();
+		int totalRow = driver.findElements(By.xpath("//table[@id='table1']/tbody/tr")).size();
+		for (int index = 1; index <= totalRow; index++) {
+			String fName = driver.findElement(By.xpath("//table[@id='table1']/tbody/tr[" + index + "]/td[2]"))
+					.getText();
+			String lName = driver.findElement(By.xpath("//table[@id='table1']/tbody/tr[" + index + "]/td[3]"))
+					.getText();
+			String convertToLower= fName.toLowerCase();
+			char c = convertToLower.charAt(0);
```

Comment:
why to create many variabels, you can handle this with single variable. think on this, refer brijesh/anjali code